src/module/helpers/processResponseData.js

```diff
@@ -1,31 +1,78 @@
 import { setResourceObjectsForModule } from './setResourceObjectsForModule'
-import { isMissingModule, registerMissingModule } from './missingModule'
 import { hasOwn } from '../../shared/utils'
+import { ResourceProxy } from '../../api/ResourceProxy'
+import { ModuleBuilder } from '../ModuleBuilder'
+
+function isAllowedRelatedModule (relatedModule, currentMethod) {
+  return relatedModule !== null &&
+    relatedModule.state.options.absoluteMethods.includes(currentMethod)
+}
+
+/**
+ * Check if a module is registered in the store
+ *
+ * @param {Vuex} store
+ * @param {String} moduleName
+ */
+function isMissingModule (store, moduleName) {
+  return Object.keys(store._modules.root._children).indexOf(moduleName) < 0
```

Comment:
shouldn't this be `<= 0` ?

Reply:
No, because `0` would be a valid array index. `indexOf` returns `-1` if a matching index cannot be found.

Reply:
never mind. You are right.